packages/main/src/components/ActionSheet/ActionSheet.stories.mdx

```diff
@@ -126,6 +126,37 @@ const MyComponentWithActionSheet = () => {
 };
 ```
 
+**Opening an `ActionSheet` by reference and not by `id`**
+
+The web component exposes a way to pass a reference of an element instead of an `id` to the `opener` prop. Since this is not supported when passing the reference in a declarative way to a React `prop`, you have to attach the ref directly on the web component.
```

Comment:
```suggestion
The underlying `ui5-responsive-popover` web component exposes a way to pass a reference of an element instead of an `id` to the `opener` prop. Since this is not supported when passing the reference in a declarative way to a React `prop`, you have to attach the ref directly on the web component.
```